recipes-domd/agl/files/meta-xt-prod-extra/recipes-connectivity/connman/files/main.conf

```diff
@@ -0,0 +1,2 @@
+[General]
+NetworkInterfaceBlacklist=eth,xenbr,vif
```

Comment:
I would put a comment here why we blacklist interfaces being in use in the system

Reply:
commit message describes why do we need this